src/app/clusters/diagnostic-logs-server/BDXDiagnosticLogsProvider.h

```diff
@@ -77,7 +77,7 @@ class BDXDiagnosticLogsProvider : public bdx::Initiator
      * This method is called to reset state. It resets the transfer, cleans up the
      * exchange and ends log collection.
      */
-    void Reset();
+    void Reset(CHIP_ERROR error);
```

Comment:
Can we do this backwards-compatibly (e.g. by having error be optional, defaulting to something)?

Also, something needs to document the meaning of the "error" argument.